run/data_module/base_seq_cls.yaml

```diff
@@ -7,4 +7,5 @@ max_length: 512
 limit_dataset_samples: null
 shuffle: true
 sequence_order: "random"
+transform_datasets: false
```

Comment:
i think this setting should be only in the final config, not the other levels
so when you are doing your global setting you decide whether to retransform or not. if it is repeated it gets confusing
```suggestion
```